### ISSUE TITLE
Spaccing grid lines hide the sprite in Autotile editor

### ISSUE THREAD

**Issue by Charlignon** [godotengine/godot#20470]
**Godot version:**
Godot Engine v3.0.5.stable.official.6a88e22

**OS/device including version:**
Windows 10

**Description:**
Sometimes autotileset sprites have lines of blank pixels to separate and help distinguish tiles. Godot allow to set a spacing in those cases. This separate the tiles as it should.
However, the turquoise guide grid also add lines on the borders of the image, and occlude the sprite behind. In consequence , all grid lines are shifted. Note that it does not affect the separation.

To sum up, there is a shift between Model and View. The main problem is that a part of the tile you areworking on is hidden, and it is painful when you add Collision or Navigation for example. 

**Pictures:**
1px spacing : http://prntscr.com/kb4il9
5px spacing : http://prntscr.com/kb4isk

**Minimal reproduction project:**
[Spacing issue.zip](https://github.com/godotengine/godot/files/2230446/Spacing.issue.zip)

**Comment by akien-mga** [comment]
CC @MarianoGnu

**Comment by MarianoGnu** [comment]
I think i corrected this on https://github.com/godotengine/godot/pull/20585
please confirm

**Comment by akien-mga** [comment]
Assuming fixed by #20585.
